x/group/keeper/keeper_test.go

```diff
@@ -2443,6 +2452,150 @@ func (s *TestSuite) TestExecProposal() {
 	}
 }
 
+func (s *TestSuite) TestExecPrunedProposalsAndVotes() {
```

Comment:
@likhita-809 I remember telling you to write a new test for pruning, but now am thinking if it's a good idea.

Is there a code path that is covered in this test, and NOT anywhere in the above tests? If no, then probably we can just delete this test.

Reply:
In the above test, we're not checking if the proposal and votes exist when executor result is success and there is no expected error msg there. In this test, we cover these things.
Should we keep it by adding the mentioned checks or delete it, wdyt? @amaurym

Reply:
Those tests are super big already, so I'm fine leaving it as is in a separate test just for pruning